### ISSUE TITLE
[BUG] ADVANCED_PAUSE_FEATURE and NOZZLE_PARK_FEATURE no longer heating bed

### ISSUE THREAD

**Issue by Mesmer88** [MarlinFirmware/Marlin#15766]
### Description

Marlin 2.0 Bugfix: when activating ADVANCED_PAUSE_FEATURE and NOZZLE_PARK_FEATURE printer won't heat up heatbed when starting a gcode print. If i order heat bed from the LCD menu it works.
Also with the 2 options heated bed crashed into the nozzle.
When removing the two options all goes back to normal

Hardware: SKR mini E3 dip v1.0 , Ender 5, TMC2208

### Steps to Reproduce

1. Activate ADVANCED_PAUSE_FEATURE and NOZZLE_PARK_FEATURE
2. Start a print

**Expected behavior:** 
Bed starts to heat then hot end

**Actual behavior:** [What actually happens]
Bed is not heating. LCD indicator shows heater element is on.

* Include a ZIP file containing your `Configuration.h` and `Configuration_adv.h` files.
* Provide pictures or links to videos that clearly demonstrate the issue.
* See [How Can I Contribute](#how-can-i-contribute) for additional guidelines.

[park_fail.zip](https://github.com/MarlinFirmware/Marlin/files/3800278/park_fail.zip)

**Comment by boelle** [comment]
@Mesmer88 i assume this is still an issue ?

**Comment by Mesmer88** [comment]
Yes, also tried updating to latest Marlin when i first reported it.
Wondering it anyone else can replicate the problem.

**Comment by boelle** [comment]
no worked fine for me, no problem with bed heating

**Comment by boelle** [comment]
@thinkyhead @shitcreek time to have a look here before the issue gets lost and not fixed

**Comment by Mesmer88** [comment]
Today i tried to retest the bug:
I upgraded to latest Marlin 2.0 bug-fix, used new config files.
Did two tests, one firmware without ADVANCED_PAUSE and one with it.

Again ADVANCED PAUSE was causing erratic behavior on the printer. This time when hitting "Print from media" the printer would halt and buzzer sounds like siren. After power cycling the printer it would not boot up.

I removed ADVANCED_PAUSE and NOZZLE_PARK, all goes back to normal.
